docs/CHANGELOG.asciidoc

```diff
@@ -79,6 +79,20 @@ This section summarizes the changes in each release.
 [[breaking-6.3.0]]
 === Breaking Changes
 
+Operations::
++
+In version 6.3.0 x-pack is bundled as a module and you do not need to install it as a plugin. 
```

Comment:
Possible change of wording

---

As of Kibana 6.3.0, X-Pack is bundled by default in the default distribution. Existing X-Pack users should remove the plugin before upgrading.

There is a known issue with 6.3.0 preventing `bin/kibana-plugin` from removing the pre-existing plugin after upgrading. This issue will be resolved in 6.3.1. If you happen to encounter this issue, there are a few options:

* Uninstall the plugin before upgrading: `bin/kibana-plugin remove x-pack`
* Manually delete the `plugins/x-pack` directory.